src/components/chart/chart.js

```diff
@@ -52,13 +55,18 @@ class HighchartsBaseChart {
             ? parseInt(this.node.dataset.highchartsYAxisTickIntervalDesktop)
             : undefined;
         this.commonChartOptions = new CommonChartOptions(this.xAxisTickIntervalDesktop, this.yAxisTickIntervalDesktop);
+        this.estimateLineLabel = this.node.dataset.highchartsEstimateLineLabel || undefined;
```

Comment:
Nitpick: `this.estimateLineLabel` and `this.uncertaintyRangeLabel` are now inconsistent in how they are declared. I guess if they default to `undefined` anyway we don't actually need the `|| undefined` for the first one.